src/include/daos_pipe.h

```diff
@@ -0,0 +1,257 @@
+/**
```

Comment:
maybe pipe is not a great term to use. would suggest to rename to daos_pipeline

Reply:
Changed